api/v1beta1/nmstate_types.go

```diff
@@ -58,6 +58,12 @@ type NMStateSpec struct {
 	// +kubebuilder:default:={}
 	// +optional
 	ProbeConfiguration NMStateProbeConfiguration `json:"probeConfiguration,omitempty"`
+	// Verbose defines the verbosity level for nmstate operations.
+	// Valid values are "production" (default, minimal output) and "debug" (verbose output for debugging).
+	// +kubebuilder:default:="production"
+	// +kubebuilder:validation:Enum=production;debug
+	// +optional
+	Verbose shared.VerboseLevel `json:"verbose,omitempty"`
```

Comment:
Let's work on the wording here. I'd rather see `verbose: true / false` (not necessarily a boolean, this is up to you and what k8s suggests) or `level: verbose`

Reply:
so this knobs affects two things
- nmstatectl verbose level (-v, -vv, -vvv)
- handler golang trace level (debug, info, warning, error)

Maybe instead of abstract things away we need a pair of knobs:
- cliVerboseLevel(1, 2, 3)
- handlerTraceLevel(debug, info, warning, error)